xarray/core/accessor_str.py

```diff
@@ -883,6 +1105,45 @@ def count(
         Returns
         -------
         counts : array of int
+
+        Examples
+        --------
+        >>> da = xr.DataArray(["jjklmn", "opjjqrs", "t-euuJJvwx"], dims="x")
+        >>> da
+        <xarray.DataArray (x: 3)>
+        array(['jjklmn', 'opjjqrs', 't-euuJJvwx'], dtype='<U10')
+        Dimensions without coordinates: x
+
+        Using a string:
+        >>> da.str.count("jj")
+        <xarray.DataArray (x: 3)>
+        array([1, 1, 0])
+        Dimensions without coordinates: x
+
+        Enable case-insensitive matching by setting case to false:
+        >>> counts = da.str.count("jj", case=False)
+        >>> counts
+        <xarray.DataArray (x: 3)>
+        array([1, 1, 1])
+        Dimensions without coordinates: x
+
+        Using regex:
+        >>> pat = "jj\\w+"
```

Comment:
It seems that the build system does not like the double escaped strings.
No idea how to solve this though, maybe we could use a raw string `pat = r"jj\w+"`?

Reply:
Ok, then I think this is a bug of our docstring parser.
The problem is that if we use this example, it will pass the CI pipeline, but it will be actually wrong?
Or how does this render in the webpage?

Maybe we can construct an example without backslashes?

Reply:
Yeah! I think I can just use [a-z] or [0-9].